salt/modules/kubernetes.py

```diff
@@ -78,6 +83,25 @@ def __virtual__():
     return False, 'python kubernetes library not found'
 
 
+class TimeoutException(Exception):
```

Comment:
Can you use one of the exceptions already defined in `salt.exceptions.py`?

Reply:
I've looked through the list of exceptions and nothing seems to really fit.

Reply:
Hrm. I am thinking that defining an exception class in the module itself might not be the best way to go moving forward.

What do you think @cachedout? Should something like this go in `salt/exceptions.py`? Is there a better name for this more general exception than the ones that are already in `salt/exceptions.py`?

Reply:
I would vastly prefer that we keep all of our defined exceptions in one place. There should be no issue at all just moving this new one over there.